README.md

```diff
@@ -229,7 +230,7 @@ You will need to add `--no-dry-run` to the following bash script to completely d
 # Delete cluster/resources once finished
 ./clean ${KF_DIR}
 ```
-### Model evaluation
+### Model evaluation - Classifier
 
 #### Running testing locally
 Download TFRecords down from GCP either change the flags in aiaia_classifier/eval.py code.
```

Comment:
Can this be pointed to the Azure blob path instead, when the data is moved from S3 to Azure?

Reply:
Yes definitely. I have a separate branch where I'm working on the azure port where I'll work on these changes. 

This PR was more to move over the evaluation  clarify dockerfile and requirements.txt purposes. 

The evaluation also has been tested locally and instructions are for local. I'd like to get this on azure too, but if it it just works locally I think that's fine too since it's not a lot to download the test TFRecords.

Reply:
👌 sounds good.